src/pages/account/[address].tsx

```diff
@@ -0,0 +1,109 @@
+import { Container, Divider, VStack, Box, Link } from '@chakra-ui/react';
+import { Trans, t } from '@lingui/macro';
+import { graphql, Link as GatsbyLink, withPrefix } from 'gatsby';
+import type { FunctionComponent } from 'react';
+import { getAddress } from 'viem';
+
+import banner from '../../assets/images/seo/home.png';
```

Comment:
This should be changed.

Reply:
can you give us more detail?

Reply:
This is the banner we use for the home page. Ideally we use a different one for addresses.